sympy/solvers/ode.py

```diff
@@ -3691,6 +3691,7 @@ def ode_2nd_power_series_regular(eq, func, order, match):
       Historical Notes", p.p 176 - 184
 
     """
+
```

Comment:
drop this?  (I think its quite common to start code immediately following the docstring)